<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>888</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Student Management&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>551</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(248, 255, 184);</string>
    </property>
    <property name="title">
     <string>List of Students:</string>
    </property>
    <widget class="QTableWidget" name="tableWidgetStudent">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>501</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 255);</string>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Age</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>561</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 255);</string>
    </property>
    <property name="title">
     <string>Student Details:</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Code:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Age:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Intro:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditId">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(253, 255, 205);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCode">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(253, 255, 205);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditName">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(253, 255, 205);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAge">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(253, 255, 205);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditIntro">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(253, 255, 205);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAvatar">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>230</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 127);</string>
     </property>
     <property name="text">
      <string>Avatar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRemoveAvatar">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>230</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 127);</string>
     </property>
     <property name="text">
      <string>Remove Avatar</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAvatar">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>221</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../images/ic_no_avatar.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>680</y>
      <width>551</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 255, 189);</string>
    </property>
    <property name="title">
     <string>Actions:</string>
    </property>
    <widget class="QPushButton" name="pushButtonNew">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 255);</string>
     </property>
     <property name="text">
      <string>New</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/ic_new.png</normaloff>../images/ic_new.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonInsert">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 255);</string>
     </property>
     <property name="text">
      <string>Insert</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/ic_save.png</normaloff>../images/ic_save.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonUpdate">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 255);</string>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/ic_update.png</normaloff>../images/ic_update.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRemove">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 255);</string>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/ic_delete.png</normaloff>../images/ic_delete.png</iconset>
     </property>
    </widget>
   </widget>
   <zorder>label</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
